boards/Kconfig

```diff
@@ -1399,7 +1417,7 @@ config ARCH_BOARD_NUCLEO_L552ZE
 
 config ARCH_BOARD_QEMU_I486
 	bool "Qemu i486 Mode"
-	depends on ARCH_X86 || ARCH_I486
+	depends on ARCH_CHIP_QEMU_I486
```

Comment:
The same change is merged https://github.com/apache/incubator-nuttx/pull/6374/files#diff-60cc096e3a9b22a769602cbbc3b0f5e7731e72db7b0338da04fcf665ed753b64R1402